File: Login-registro/js/Usuario.js
import { SimularBd } from "./SimularBd.js";
export class Usuario{
    #id;
    #nombre;
    #correo;
    #contraseña;
    constructor(Id='',Nombre='',Correo='',Contraseña=''){
        this.#id = Id;
        this.#nombre = Nombre;
        this.#correo = Correo;
        this.#contraseña = Contraseña;
    }
    getId(){
        return this.#id;
    }
    setId(Id){
        this.#id = Id;
    }
    getNombre(){
        return this.#nombre;
    }
    setNombre(Nombre){
        this.#nombre = Nombre;
    }
    getCorreo(){
        return this.#correo;
    }
    setCorreo(Correo){
        this.#correo = Correo;
    }
    getContraseña(){
        return this.#contraseña;
    }
    setContraseña(Contraseña){
        this.#contraseña = Contraseña;
    }
    IniciarSesion(correo, contraseña){
        
    }
    //Usaremos el operador ...
    Registrarse(Nombre,Correo,Contraseña,ConfirmarC){
        let V = [];
        if(Contraseña==ConfirmarC){
            console.log(SimularBd.insertDatos(V.push(Nombre,Correo,Contraseña)));
        }else{
            alert("Las contraseñas no coinciden");
        }
    }
}

File: Clases Java/Usuario.java
public class Usuario {
    private Integer id;
    private String nombre;
    private String correo;
    private String contraseña;
    public Usuario(Integer id, String nombre, String correo, String contraseña){
        this.id = id;
        this.nombre = nombre;
        this.correo = correo;
        this.contraseña = contraseña;
        boolean valor = false;
        for(int i = 0; i<correo.length();i++){
            if(correo.charAt(i)=='@') {
                valor = true;
            }
        }
        if(!valor){
            System.out.println("Ingrese @");
        }
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getContraseña() {
        return contraseña;
    }

    public void setContraseña(String contraseña) {
        this.contraseña = contraseña;
    }

    public String[] iniciarSesion(String correo, String contraseña, String Ccontraseña){
        String[] vector = {};
        if(contraseña != Ccontraseña){
            System.out.println("Las contraseñas no coinciden");
        }else{
            vector[0]=correo;
            vector[1]=contraseña;
        }
        return vector;
    }

    public void registrase(){
        
    }
}